### PR TITLE
Session02 homework

### PR DESCRIPTION
**Pull request by rushinstuffin**
Here are the three files for homework this week. Let me know if there are any issues, please!

### CODE REVIEW

File: students/KyleBarry/session02/fizzbuzz.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+def fizz_buzz():
```
Comment (PythonCHB) [inline students/KyleBarry/session02/fizzbuzz.py]: good solid solution. If you want a bit of a challenge, see if you you refactor it to not compute % 3 and % 5 twice.

File: students/KyleBarry/session02/grid_printer.py [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    
+    plus_line = plus + (hyphen*4) + ' ' + plus + (hyphen*4) + ' ' + plus + '\n'
+    no_plus_line = '|' + empties*4 + ' ' + '|' + empties*4 + ' ' + '|' + '\n'
+    
+    print(plus_line + (no_plus_line*4) + plus_line +
+    (no_plus_line*4) + plus_line)
+
+def grid_adjusted(n):
+    """Part II: adjust the size of the grid"""
+    plus = '+'
```
Comment (PythonCHB) [inline students/KyleBarry/session02/grid_printer.py]: not sure there is much point in using a variable for a single charater. unless you are thinking that you might want to change it to a different character -- if that's the case, then use a name that makes sense if you change it -- like "corner" maybe?

File: students/KyleBarry/session02/grid_printer.py [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    empties = '  '
+
+    plus_line = plus + (hyphen*c) + ' '
+    no_plus_line = '|' + (empties*c) + ' '
+
+    full_lines = plus_line*d + plus + '\n'
+    next_lines = no_plus_line*d + '|' + '\n'
+
+    alls = full_lines+next_lines*c
+    print(alls*d + full_lines)
```
Comment (PythonCHB) [inline students/KyleBarry/session02/grid_printer.py]: nicely done with no looping!

File: students/KyleBarry/session02/series.py [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+
+if __name__ == '__main__':
+
+    #test that fibonnaci sequencing is accurate
+    assert fibonacci(9) == 34
+    #test that lucas number sequencing is accurate
+    assert lucas(5) == 11
+
+    #test that sum_series with default params return fibonacci sequence
+    assert sum_series(4289) == fibonacci(4289)
```
Comment (PythonCHB) [inline students/KyleBarry/session02/series.py]: that's a pretty big number!

File: students/KyleBarry/session02/series.py [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+
+    #test that sum_series with lucas number starting params returns lucas
+    #numbers sequence
+    assert sum_series(5, 2, 1) == lucas(5)
+    assert sum_series(45, 2, 1) == lucas(45)
+    print('All assertions passed!')
```
Comment (PythonCHB) [inline students/KyleBarry/session02/series.py]: nice asserts! you are on your way to unit testing....

**Comment by rushinstuffin** [comment]
Thanks for all the feedback here. Super helpful!

**Pull request merged**